docs/devguide/xtypes.rst

```diff
@@ -820,6 +820,35 @@ The ``@key`` annotation marks a member used to determine the Instances of a topi
 See :ref:`getting_started--keys` for more details on the general concept of a Key.
 For XTypes specifically, two types can only be compatible if each contains the members that are keys within the other.
 
+Customizing the values of enumerators
+=====================================
+
+..
+    Sect<16.6.7>
```

Comment:
```suggestion
```

The section comments were for debugging the conversion script, but I kept in case someone wanted to search using an old section reference. I don't think we need to add numbers for new sections.

Reply:
Of course I just copied and pasted the section above it.  I'll update this.